# This is a message to define a goal pose of a MM-UAV

std_msgs/Header header

geometry_msgs/Pose pose #pose of the UAV base
std_msgs/Float64[] joints #joint position in radians
